airflow/jobs.py

```diff
@@ -1185,7 +1185,13 @@ def _process_dags(self, dagbag, dags, tis_out):
 
             dag_run = self.create_dag_run(dag)
             if dag_run:
-                self.logger.info("Created {}".format(dag_run))
+                expected_start_date = dag.following_schedule(dag_run.execution_date)
+                if expected_start_date:
+                    schedule_delay = dag_run.start_date - expected_start_date
+                    Stats.timing(
+                        'dagrun.schedule_delay.{dag_id}'.format(dag_id=dag.dag_id),
+                        schedule_delay)
+                self.log.info("Created %s", dag_run)
```

Comment:
what's the difference here between `self.log.info` and `self.logger.info`?

Reply:
Actually, we should use `self.logger`.